apps/intellij-plugin/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ repositories {
 // Configure Gradle IntelliJ Plugin
 // Read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
 intellij {
-    version.set("2022.3.2")
+    version.set("2023.1")
```

Comment:
Is this everything, which is needed to update to the latest version? (just for clarification for the future)

Reply:
This is just the version for the debug run configuration, the actual version for the plugin is set down below and has not been adjusted. I can revert to the old version; this is a remnant of testing.

Reply:
```suggestion
    version.set("2022.3.2")
```